content/03.text-mining.md

```diff
@@ -0,0 +1,49 @@
+## Relationship Extraction
+
+1. Set up the context for relationship extraction
+	1. Define the problem
+	2. Talk about the importance of the problem (filling knowledge bases -> point researchers to relevant papers)
+2. Give overview towards taxonomy of approaches (hand written rules, unsupervised machine learning, supervised machine learning etc.)
+
+### Text Mining
+
+#### Hand written Natural Language Processing Techniques
```

Comment:
Are these relevant to biomedical knowledge graph extraction?

Reply:
Or - wait - do you mean of hand written text, or do you mean like human-written rules? If so, I might define these as:
> Rule-based NLP

Reply:
Meant human-written rules. Have made a change according to the above comment.